src/lib.rs

```diff
@@ -534,4 +534,123 @@ impl<'a, 'b> Builder<'a, 'b> {
 
         util::create_helper(dir, self.prefix, self.suffix, self.random_len, dir::create)
     }
+
+    /// Attempts to create a temporary file (or file-like object) using the
+    /// provided closure inside of [`std::env::temp_dir()`]. The closure is
```

Comment:
```suggestion
    /// provided closure inside of creating a [`std::fs::File`]. The closure is
```

Reply:
That wording still looks a little odd (`s/inside/instead/`?). I'll try to make this clearer.

Reply:
Uh, yes.

Reply:
Oh, wait, nvm. I just read this as "instead of". No, the current wording makes sense. Although re-ordering to it as "temporary file inside of ... with the provided closure" may be easier (at least for me).

Reply:
I reworded this. It should be clearer now.